src/daemon/demo.sh

```diff
@@ -58,6 +58,12 @@ if [[ "$RGW_FRONTEND_TYPE" == "beast" ]]; then
 fi
 
 
+function fetch_initial_keys {
+  for key in $(ceph -n mon. -k /var/lib/ceph/mon/ceph-mon0/keyring auth list |& grep ^client); do
```

Comment:
why mon0 ? isn't like a little bit of hardcoding here ?

Reply:
Yeah this is wrong :)